test/testdata/lsp/fast_path/linearization.1.rbupdate

```diff
@@ -0,0 +1,26 @@
+# typed: true
```

Comment:
fwiw i looked back and it looks like this test has failed for at least the past 6 months (i was curious to know whether it was a recent regression, and it doesn't look like it). Nice find!